metadata.js

```diff
@@ -1,4 +1,12 @@
 PostList = [
-  {title: 'A Demo is Worth a Thousand Words', template: "a_demo"},
+  {
+    title: "A Demo is Worth a Thousand Words",
+    template: "a_demo",
+    excerpt: "Last year using an older version of the Universal Recommender I built a little demo app. It runs in my closet on an old desktop over Comcast so don’t...",
+    author: "Pat Ferrel",
+    date: "Apr 1, 2016",
+    thumb: "/images/Guide-front-1024x7211.png",
+    categories: ["use case", "recommender"]
+  },
   {title: 'Second blog post', template: "second_blog_post"},
```

Comment:
I thought we were going to reference a template inside the post so a post would contain a template for the post and a snippet for the blog front. We should be able to do this with the markdown template extension and would be a greatly improved workflow. Adding the templated to metadata.js is fine but we would not put content in metadata.

I thought is was @pote who suggested this?

I'll pull this in any case as a first step.